web-admin/src/components/projects/ProjectDeploymentLogs.svelte

```diff
@@ -22,16 +24,22 @@
 </script>
 
 {#if $proj.isSuccess && errors}
-  {#if errors.length === 0}
-    <p class="text-gray-500 my-6">No logs available.</p>
-  {:else}
-    <!-- logs count -->
-    <div class="w-full px-12 py-3 border-b border-gray-200 font-semibold">
-      <span class="text-red-600">{errors.length} </span>
-      <span class="text-gray-800"> error(s)</span>
-    </div>
-    <!-- logs -->
-    <ul class="w-full">
+  <ul class="w-full">
+    {#if !hasAdminAccess}
+      <li class="px-12 py-2 font-semibold text-gray-500 border-b">
+        You don't have permission to view project logs
+      </li>
+    {:else if errors.length === 0}
+      <li class="px-12 py-2 font-semibold text-gray-500 border-b">
+        No logs present
+      </li>
```

Comment:
These aren't really "list items," so I'd move this code up above the `<ul>` tag.

Reply:
The mocks make it look like a list with a single element so I would prefer to have it this way. Also it makes the component structure much simpler.